### ISSUE TITLE
Local dev broken since helix-content-proxy

### ISSUE THREAD

**Issue by rofe** [adobe/helix-pipeline#792]
**Description**
In my local development environment for https://github.com/adobe/theblog, `helix-pipeline` v9+ (since the introduction of `helix-content-proxy`) doesn't work for me, i.e. keeps serving up 404s for any blog URLs. The weird thing is when I request the proxy url reported by `hlx` in the console - either using curl or the browser - the page that was previously not found suddenly renders fine. But the very next reload will result in a 404 again.

**To Reproduce**
1. `git clone https://github.com/adobe/theblog.git`
2. `cd theblog`
3. `hlx up`
4. Request `http://localhost:3000/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.html` > 404
5. Copy content proxy URL from the output and request it in the browser: `https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fpublish%2F2020%2F06%2F17%2F5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md&ref=issues-241&ignore=github` > shows markdown 
6. Reload `http://localhost:3000/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.html` > works! (well, for this particular request, but then fails again when the browser tries to load the author info)

**Expected behavior**
The above URL should always render a blog post.

**Actual behavior**
The above URL is not found.

- what's the output you are seeing?
404
- are there error messages?
```
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:61650/raw/helix/github.com--adobe--theblog/issues-241/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md
[hlx] info: content proxy: http://127.0.0.1:61650/raw/helix/github.com--adobe--theblog/issues-241/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md does not exist. 404 not found.
[hlx] info: content proxy: proxy to https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fpublish%2F2020%2F06%2F17%2F5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md&ref=issues-241&ignore=github
[hlx] info: Could not find file at http://localhost:3000/__internal__/content-proxy/default?owner=helix&repo=github.com--adobe--theblog&path=%2Fen%2Fpublish%2F2020%2F06%2F17%2F5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md&ref=issues-241&REPO_RAW_ROOT=http%3A%2F%2F127.0.0.1%3A61650%2Fraw%2F
[hlx] warn: Could not find file at http://127.0.0.1:61650/raw/helix/github.com--adobe--theblog/issues-241/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md
[hlx] warn: Problems while executing action: Error while fetching file from http://127.0.0.1:61650/raw/helix/github.com--adobe--theblog/issues-241/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md: 404
[hlx] error: Resource not found: /en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.html
```

**Version:**
helix-cli@11.0.1
helix-pages@2.7.2
helix-pipeline@10.1.1
helix-simulator@4.3.5

**Additional context**
Workaround:
1. stop `hlx up`
2. `cd .hlx/pages/master`
3.  in `package.json`, change `helix-pipeline` version to `8.0.7`
4. `npm i`
5. `cd ../../..`
6. `hlx up`

**Comment by kptdobe** [comment]
Here are my logs:

```
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md does not exist. 404 not found.
[hlx] info: content proxy: proxy to https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fpublish%2F2020%2F06%2F17%2F5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md&ref=master&ignore=github
[hlx] info: source-location: /drives/b!PpnkewKFAEaDTS6slvlVjh_3ih9lhEZMgYWwps6bPIWZMmLU5xGqS4uES8kIQZbH/items/01DJQLOWY5OQ4D2FSXGJDYXBGXPZHXAE27
[hlx] info: source-hash: 3oBelJZjvIRJAG03
[hlx] warn: Error: Found ambiguous frontmatter fence: Block contains empty line! Make sure your mid-document YAML blocks contain no empty lines and your horizontal rules have an empty line before AND after them.

by Kaveri Misra

posted on 06-17-2020

    at warn (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/@adobe/helix-pipeline/src/html/remark-matter.js:35:17)
    at Of.frontmatter ./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/@adobe/helix-pipeline/src/html/remark-matter.js:96:14)
    at Of.tokenize [as tokenizeBlock] (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/remark-parse/lib/tokenizer.js:62:18)
    at Of.parse (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/remark-parse/lib/parse.js:33:20)
    at Function.parse (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/unified/index.js:281:45)
    at parseMarkdown (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/@adobe/helix-pipeline/src/html/parse-markdown.js:37:6)
    at execFns (./work/dev/helix/theblog/theblog/.hlx/pages/master/node_modules/@adobe/helix-pipeline/src/pipeline.js:392:17)
    at process._tickCallback (internal/process/next_tick.js:68:7)
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/header.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/footer.md
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md does not exist. 404 not found.
[hlx] info: Could not find file at http://localhost:3000/__internal__/content-proxy/default?owner=helix&repo=github.com--adobe--theblog&path=%2Fhead.md&ref=master&REPO_RAW_ROOT=http%3A%2F%2F127.0.0.1%3A49191%2Fraw%2F
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/header.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/footer.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] warn: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: Writing context dumps to: ./work/dev/helix/theblog/theblog/logs/debug/context_dump_20200607-1157-47.0100_head.html_1c96438444bd3d6f483661e4dc7d1eb4
[hlx] warn: Problems while executing action: Error while fetching file from http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md: 404
[hlx] info: no script for css
[hlx] info: no script for js
[hlx] info: no script for js
[hlx] info: no script for css
[hlx] info: no script for js
[hlx] info: no script for json
[hlx] info: no script for ico
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md does not exist. 404 not found.
[hlx] info: content proxy: proxy to https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fauthors%2Fkaveri-misra.md&ref=master&ignore=github
[hlx] info: source-location: /drives/b!PpnkewKFAEaDTS6slvlVjh_3ih9lhEZMgYWwps6bPIWZMmLU5xGqS4uES8kIQZbH/items/01DJQLOWZE7YJCQ5YBRBAJ7FONHRTYP47C
[hlx] info: source-hash: sid10Y7RJzEGIj+S
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/footer.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/header.md
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md does not exist. 404 not found.
[hlx] info: Could not find file at http://localhost:3000/__internal__/content-proxy/default?owner=helix&repo=github.com--adobe--theblog&path=%2Fhead.md&ref=master&REPO_RAW_ROOT=http%3A%2F%2F127.0.0.1%3A49191%2Fraw%2F
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/footer.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/header.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] warn: Could not find file at http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: Writing context dumps to: ./work/dev/helix/theblog/theblog/logs/debug/context_dump_20200607-1157-49.0272_head.html_df45225bbe97f72aec30576d158ae615
[hlx] warn: Problems while executing action: Error while fetching file from http://127.0.0.1:49191/raw/helix/github.com--adobe--theblog/master/head.md: 404
```

**Comment by rofe** [comment]
I do get the same or very similar output after doing step 5. Here's my console output from step 6:
```
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md
[hlx] info: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/publish/2020/06/17/5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md does not exist. 404 not found.
[hlx] info: content proxy: proxy to https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fpublish%2F2020%2F06%2F17%2F5-potential-pitfalls-to-avoid-as-you-prioritise-digital-transformation-goals.md&ref=master&ignore=github
[hlx] info: source-location: /drives/b!PpnkewKFAEaDTS6slvlVjh_3ih9lhEZMgYWwps6bPIWZMmLU5xGqS4uES8kIQZbH/items/01DJQLOWY5OQ4D2FSXGJDYXBGXPZHXAE27
[hlx] info: source-hash: 3oBelJZjvIRJAG03
[hlx] warn: Error: Found ambiguous frontmatter fence: (...)
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/header.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/footer.md
[hlx] info: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/head.md does not exist. 404 not found.
[hlx] info: Could not find file at http://localhost:3000/__internal__/content-proxy/default?owner=helix&repo=github.com--adobe--theblog&path=%2Fhead.md&ref=master&REPO_RAW_ROOT=http%3A%2F%2F127.0.0.1%3A50361%2Fraw%2F
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/header.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/footer.md: 200
[hlx] info: source-location: 
[hlx] info: source-hash: 2jmj7l5rSw0yVb/v
[hlx] info: unable to fetch helix-markup.yaml: 404
[hlx] warn: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/head.md
[hlx] info: Writing context dumps to: /Users/rofe/code/theblog/logs/debug/context_dump_20200607-1216-16.0919_head.html_43b49ac6a77a1479ff0af24e46c5c169
[hlx] warn: Problems while executing action: Error while fetching file from http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/head.md: 404
[hlx] info: no script for css
[hlx] info: no script for js
[hlx] info: no script for js
[hlx] info: no script for js
[hlx] info: no script for css
[hlx] info: no script for json
[hlx] info: no script for ico
[hlx] info: content proxy: try loading from github first: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md
[hlx] info: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/helix-markup.yaml
[hlx] info: content proxy: http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md does not exist. 404 not found.
[hlx] info: content proxy: proxy to https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1?owner=adobe&repo=theblog&path=%2Fen%2Fauthors%2Fkaveri-misra.md&ref=master&ignore=github
[hlx] info: no script for ico
[hlx] info: no script for ico
[hlx] info: Could not find file at http://localhost:3000/__internal__/content-proxy/default?owner=helix&repo=github.com--adobe--theblog&path=%2Fen%2Fauthors%2Fkaveri-misra.md&ref=master&REPO_RAW_ROOT=http%3A%2F%2F127.0.0.1%3A50361%2Fraw%2F
[hlx] warn: Could not find file at http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md
[hlx] info: Writing context dumps to: /Users/rofe/code/theblog/logs/debug/context_dump_20200607-1216-18.0499_en-authors-kaveri-misra.html_8101bb91160f9d8bbec988e938eb59f1
[hlx] warn: Problems while executing action: Error while fetching file from http://127.0.0.1:50361/raw/helix/github.com--adobe--theblog/master/en/authors/kaveri-misra.md: 404
[hlx] error: Resource not found: /en/authors/kaveri-misra.html
```

**Comment by rofe** [comment]
Fixed in @helix-cli@11.0.3
